src/models/Airline.ts

```diff
@@ -0,0 +1,357 @@
+                                media: hasTakeoffScreenshot ? {media_ids: [mediaId]} : undefined
+                            })
+                        }
+                    }
+                    resolve(true);
+                    break;
+                case PlaneEvents.PLANE_LAND:
+                    let hasLandingScreenshot = false;
+                    if(this.notifications.includeScreenshots){
+                        hasLandingScreenshot = await Common.takeScreenshot(aircraft.icao);
+                    }
+                    if (this.notifications.discord.enabled) {
+                        let message = `**${this.name}** flight ${aircraft.callsign} (${aircraft.registration}) landed at **${data.nearestAirport.name}** at <t:${(new Date().getTime() / 1000).toFixed(0)}:t>\n${adsbExchangeLink}`;
+
+                        for (const discord of this.notifications.discord.webhooks) {
+                            const hook = new WebhookClient({url: discord});
+                            PlaneAlert.log.debug(`Plane ${this.name} (${aircraft.icao}) sending discord notification to ${discord}`);
+                            if (hasLandingScreenshot) {
+                                // Get RawFile of `/tmp/${aircraft.icao}.png`
+                                const rawFile = await fs.readFileSync(`/tmp/${aircraft.icao}.png`);
+                                await hook.send({
+                                    username: this.name + ' - ' + aircraft.registration,
+                                    avatarURL: photoUrl ?  photoUrl : null,
+                                    content: message,
+                                    files: [rawFile]
+                                });
+                            }
+
+                        }
+                    }
+                    if (this.notifications.twitter.enabled) {
+                        for(const account of this.notifications.twitter.accounts) {
+                            const client = TwitterUtils.getTwitterClient(account.accessToken, account.accessSecret);
+                            let mediaId = '';
+                            if(hasLandingScreenshot){
+                                mediaId = await client.v1.uploadMedia(`/tmp/${aircraft.icao}.png`);
+                            }
+                            await client.v2.tweet({
+                                text: `${this.name} (${aircraft.callsign}) (#${aircraft.registration}) landed at ${data.nearestAirport.name} at ${new Date().toLocaleString()}\n${adsbExchangeLink}`,
+                                media: hasLandingScreenshot ? {media_ids: [mediaId]} : undefined
+                            })
+                        }
+                    }
+                    resolve(true);
+                    break;
+            }
+        });
+    }
+
+    private async getPhotoUrl(aircraft: AircraftMeta) {
```

Comment:
Since there is a new class now (not just Aircraft), this probably should be offloaded to an external helper class